src/cli/cli.cpp

```diff
@@ -3716,6 +3717,8 @@ void Interpreter::HandleDiagnosticGetResponse(Message &aMessage, const Ip6::Mess
     uint16_t bytesPrinted = 0;
     uint16_t length       = aMessage.GetLength() - aMessage.GetOffset();
 
+    VerifyOrExit(mServer != NULL);
```

Comment:
I don't think this case is possible. This is a CoAP response handler, which requires a CoAP request to be in the past.